devTools/phpstan-baseline.neon

```diff
@@ -1865,3 +1895,9 @@ parameters:
 			identifier: missingType.return
 			count: 1
 			path: ../tests/phpunit/TestFramework/PhpUnit/Config/XmlConfigurationVersionProviderTest.php
+
+		-
+			message: '#^Unable to resolve the template type TTake in call to function Pipeline\\take$#'
+			identifier: argument.templateType
+			count: 2
+			path: ../tests/phpunit/TestFramework/PhpUnit/Config/XmlConfigurationVersionProviderTest.php
```

Comment:
Bunch of missing type annotations. But if I add them, it notices a bunch of other places, too, lack inline documentation. And it keeps exploding, sadly.

The good thing, if we'd want, we can make it all fully typed.